### PR TITLE
Tomers' constraints

### PR DESCRIPTION
**Pull request by JonatanMilver**

### CODE REVIEW

File: Booking.java [hunk 1]
```diff
@@ -68,6 +70,40 @@ public static boolean checkAllIntancesConstraints(Model model){
...
+    /**
+     * RoomCatagory type have to be in the same level or higher
+     * @return
+     */
+    public boolean constraint_8(){
+        RoomCategory.RoomType RoomTypeOfReservation = this.getReservation().getRoomCategory().getType();
+        RoomCategory.RoomType RoomTypeOfRoom = this.getRoom().getRoomCategory().getType();
+        boolean boolToReturn = true;
+        if(RoomTypeOfReservation == RoomCategory.RoomType.VIP){
+            if(!(RoomTypeOfRoom == RoomCategory.RoomType.VIP)){
```
Comment (GalAgas) [inline Booking.java:82]: why do you hat the sign !=   ?

File: Hotel.java [hunk 2]
```diff
@@ -83,4 +84,23 @@ public boolean constraint_6(){
...
+     * @return
+     */
+    public boolean constraint_11(){
+        HashMap<Service, HotelService> list_of_Services = this.getServices();
+        boolean boolToReturn = true;
+        for (Service s1: list_of_Services.keySet()) {
+            for (Service s2: list_of_Services.keySet()) {
+                if(s1 != s2){
+                    if(!(s1.getServiceName().equals(s2.getServiceName()))){
+                        boolToReturn = false;
```
Comment (JonatanMilver) [inline Hotel.java:99]: you check if two service names are different then you set boolToReturn -> false.
it should be - if two service names are equal then u set boolToReturn -> false..
am i wrong?

Comment (tomerkel) [reply Hotel.java:99]: you right.
delete the "!"

File: Booking.java (hunk 1 above)
Comment (tomerkel) [reply Booking.java:82]: believe me.. this ron ....

File: Hotel.java (hunk 2 above)
Comment (GalAgas) [reply Hotel.java:99]: you are in love with the sign !

**Pull request merged**